index.js

```diff
@@ -117,27 +125,27 @@ class SnackbarComponent extends Component {
    * @return {null} No return.
    */
   hideSnackbar() {
-    Animated.timing(
-      this.state.translateValue,
-      {
-        duration: duration_values.exit,
-        toValue: 0,
-        easing: easing_values.exit,
-      },
-    ).start();
+    Animated.timing(this.state.translateValue, {
+      duration: durationValues.exit,
+      toValue: 0,
+      easing: easingValues.exit,
+    }).start();
   }
-
 }
 
 SnackbarComponent.defaultProps = {
   accentColor: 'orange',
   messageColor: '#FFFFFF',
   backgroundColor: '#484848',
   distanceCallback: noop,
+  actionHandler: noop,
   left: 0,
   right: 0,
   bottom: 0,
+  visible: false,
   position: 'bottom',
+  actionText: '',
```

Comment:
this change is the cause of #36. Please test refactors like this properly...

Reply:
If its the cause then setting it to require would-be what's needed but at default, it wasn't even included which is another bug